### PR TITLE
Change thing's service development port

### PR DESCRIPTION
**Pull request by esdrasjnr**
What does this implement/fix? Explain your changes
---------------------------------------------------
This patch changes the development port for things service to 8182, which is configured in yaml file for the new core stack.

Where has this been tested?
---------------------------
**Operating System/Platform:** Ubuntu 18.04.4
**Go Version:** go1.13.8

### CODE REVIEW

File: internal/config/development.yaml [hunk 1]
```diff
@@ -13,4 +13,4 @@ rabbitmq:
 
 things:
   hostname: localhost
-  port: 8082
+  port: 8182
```
Comment (netoax) [inline internal/config/development.yaml]: What do you think about removing `things` and `users` configuration from the `development.yaml`? In practice, we don't need them to be defined here and it just adds one more concern to us.

Comment (netoax) [reply internal/config/development.yaml]: The developer can define them to their local usage.

Comment (esdrasjnr) [reply internal/config/development.yaml]: Done!

**Review by netoax — APPROVED** [review]

**Review by lcbm — APPROVED** [review]
Thanks for the fix :smiley:

**Pull request merged**